sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +472,15 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ProcessingContext
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public System.Exception? LastException { get { throw null; } set { } }
+        public System.DateTimeOffset ProcessingStartTime { get { throw null; } set { } }
```

Comment:
Do we need the `Processing` at the beginning of `StartTime` if it is on a type called `ProcessingContext`?

Reply:
Will change